tools/challenge-editor/client/src/components/tools/Tools.tsx

```diff
@@ -27,7 +27,7 @@ const Tools = () => {
       <CreateEmptySteps {...{ superblock, block }} />
       <h2>Insert Step</h2>
       <p>
-        This tool inserts a new step after the <code>nth</code> step.
+        This tool inserts a step as the <code>nth</code> step.
```

Comment:
```suggestion
        This tool inserts a new step as the <code>nth</code> step.
```
I think "new" here worked, didn't it?